README.md

```diff
@@ -0,0 +1,59 @@
+# express-gateway-plugin-rewrite
+
+This plugin for [Express Gateway](https://express-gateway.io) makes it possible to redirect (rewrite internally or
+redirect using HTTP codes) User to the specific URL based on
+[Express Paths](https://expressjs.com/en/guide/routing.html) or RegExp Rules.
+
+## Installation
+
+Simply type from your shell environment:
+
+```bash
+eg plugin install express-gateway-plugin-rewrite
+```
+
+## Quick start
+
+1. Make sure the plugin is listed in [system.config.yml file](https://www.express-gateway.io/docs/configuration/system.config.yml/).
+This is done automatically for you if you used the command above.
+
+2. Add the configuration keys to [gateway.config.yml file](https://www.express-gateway.io/docs/configuration/gateway.config.yml/).
+
+```yaml
+policies:
+  -
+    rewrite:
+      -
+        condition:
+          name: pathmatch
+          match: /tina/:code
+        action:
+          rewrite: /status/:code
+          redirect: permanent
```

Comment:
Why not have 301,302 , 307 etc here ?

Reply:
Good point - I honestly ignored all the other status codes as they felt "unused" to me. I'll have a quick look to the spec again and modify the code accordingly.